gradient/cli/notebooks.py

```diff
@@ -24,72 +24,83 @@ def notebook_metrics():
     pass
 
 
-# @notebooks_group.command("create", help="Create new notebook")
-# @click.option(
-#     "--vmTypeId",
-#     "vm_type_id",
-#     type=int,
-#     required=True,
-#     help="Type of Virtual Machine",
-#     cls=common.GradientOption,
-# )
+@notebooks_group.command("create", help="Create new notebook")
 @click.option(
-    "--containerId",
-    "container_id",
-    type=int,
+    "--clusterId",
+    "cluster_id",
+    type=str,
     required=True,
-    help="Container ID",
+    help="Cluster ID",
     cls=common.GradientOption,
 )
 @click.option(
-    "--clusterId",
-    "cluster_id",
+    "--vmTypeLabel",
+    "vm_type_label",
+    type=str,
+    help="Virtual Machine type label e.g. P5000",
+    cls=common.GradientOption,
+)
+@click.option(
+    "--vmTypeId",
```

Comment:
I'm not sure if this is useful. Isn't vmTypeId an id in the database? I mean it's not shared anywhere and user has no way to acquire this unless has direct access to our database. (I'm not sure though)